src/cloudflare/internal/vectorize-api.ts

```diff
@@ -240,7 +243,7 @@ class VectorizeIndexImpl implements Vectorize {
 
 function isVectorizeMetadataRetrievalLevel(
   value: unknown
-): value is VectorizeMetadataRetrievalLevel {
+): boolean {
```

Comment:
Is there a reason for this change? I think it makes sense to keep it as it was

Reply:
I changed this to get rid of a warning `Invalid type "never" of template literal expression.eslint[@typescript-eslint/restrict-template-expressions](https://typescript-eslint.io/rules/restrict-template-expressions)`